### PR TITLE
Improve lazy loading of Monaco editor and fix webworker warning

### PR DESCRIPTION
**Pull request by martin-fleck-at**
- Load web worker through blob URL
- Avoid scripts from being loaded automatically if they are dynamic

Requires https://github.com/axonivy/process-editor-client/pull/515

### CODE REVIEW

File: extension/src/base/build-manifest.ts
Comment (ivy-lli) [inline extension/src/base/build-manifest.ts:1]: cool thanks!

**Review by ivy-lli — APPROVED** [review]
Looks good to me. @ivy-lmu ?

**Comment by ivy-lli** [comment]
@martin-fleck-at new version of the process editor packages should be in the npm repo

**Comment by martin-fleck-at** [comment]
@ivy-lli Thank you, I'll update this PR very soon!

**Review by ivy-lmu — APPROVED** [review]
👍 thanks

**Comment by ivy-lli** [comment]
@martin-fleck-at Did you update this PR? Because it seems to work now :) Can you maybe update the @axonivy dependencies in the package.json/.lock?

**Comment by ivy-lmu** [comment]
@ivy-lli I have triggered the jobs

**Comment by martin-fleck-at** [comment]
@ivy-lli @ivy-lmu Unfortunately, my internet was out for some time. I pushed the update now, sorry for the delay!

**Comment by martin-fleck-at** [comment]
I only updated the `yarn.lock`, the rest is still the same.

**Comment by ivy-lli** [comment]
The PR looks good and the builds are happy. I will merge :)

**Pull request merged**